index.css

```diff
@@ -0,0 +1,186 @@
+body {
+    background-color: #f2f2f2;
+    color: #333;
+    font-weight: 400;
+    line-height: 1.3;
+    margin: 32px;
+    font-family: Arial, Arial, Helvetica, sans-serif;
+}
+
+img {
+    max-width: 100%;
+}
+
+.nav {
+    box-sizing: border;
```

Comment:
1. `border` is not a valid value for this property; you want `border-box`;
2. Instead of using this on individual elements, consider applying `box-sizing` to the entire stylesheet; see my [best practices list](https://mica-web.github.io/learn/css/best-practices#declare-box-sizing-in-all-your-stylesheets).